### ISSUE TITLE
Accordion: Keyboard operation does not work with screen reader

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#1477]
https://w3c.github.io/aria-practices/examples/accordion/accordion.html:

> Down Arrow: When focus is on an accordion header, moves focus to the next accordion header. When focus is on last accordion header, moves focus to first accordion header.
> Up Arrow: When focus is on an accordion header, moves focus to the previous accordion header. When focus is on first accordion header, moves focus to last accordion header.
> Home: When focus is on an accordion header, moves focus to the first accordion header.
> End: When focus is on an accordion header, moves focus to the last accordion header.

Of course, these 4 key do not work if a screen reader is used, because it is not in application or forms mode, but the virtual cursor is on. The only unsightly solutions I can think of are
* use role=application or
* use other keys that are not used by the screen reader or
* on the page at least document that the keyboard operation does not work with screen reader

Is there a better solution?

**Comment by mcking65** [comment]
@JAWS-test 
We are removing the optional arrow key behavior in #1830.
Issues discussing the decision are linked from the PR.
